controller/controller_server/csi_controller_server.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, array_endpoint):
         with open(path, 'r') as yamlfile:
             self.cfg = yaml.safe_load(yamlfile)  # TODO: add the following when possible : Loader=yaml.FullLoader)
 
-    @handle_common_exceptions(csi_pb2.CreateVolumeResponse)
+    @exception_handler.handle_common_exceptions(csi_pb2.CreateVolumeResponse)
```

Comment:
I actually like the previous way better - is there a particular reason for this change?

Reply:
Fewer import statements. I changed it